bundles/org.openhab.binding.sonos/src/main/java/org/openhab/binding/sonos/internal/handler/ZonePlayerHandler.java

```diff
@@ -93,7 +93,8 @@ public class ZonePlayerHandler extends BaseThingHandler implements UpnpIOPartici
     private static final String RADIO_URI = "x-sonosapi-radio:";
     private static final String RADIO_MP3_URI = "x-rincon-mp3radio:";
     private static final String RADIOAPP_URI = "x-sonosapi-hls:radioapp_";
-    private static final String OPML_TUNE = "http://opml.radiotime.com/Tune.ashx";
+    private static final String OPML_TUNE_HTTP = "http://opml.radiotime.com/Tune.ashx";
+    private static final String OPML_TUNE_HTTPS = "https://opml.radiotime.com/Tune.ashx";
```

Comment:
Any reason you can just match `://opml.radiotime.com/Tune.ashx` and have one variable ?

Reply:
I hesitated between the two options.

Reply:
Changed